### ISSUE TITLE
always Hide nsec upon paste

### ISSUE THREAD

**Issue by alltheseas** [damus-io/notedeck#508]
- [ ] Hide nsec after paste
- [ ] Allow to manually optionally show/hide nsec with eye icon after paste

**Comment by kuba-04** [comment]
hey @alltheseas, I started looking at this but while testing it, I logged out and can't start the app again. I think there is something wrong with the initial logic when we are fetching the account keys from the local file storage. It kind of assumes we have it there otherwise we raise the error, while instead of the error we might go another path (like starting from scratch). Have you seen an issue already open for this? if not, I can open a separate issue if you agree there is a bug, or maybe I misunderstand something here!

Below I would rather have Option with None when the file doesn't exists, but the function is being used in many places so I will rather ask what you think! 

![Image](https://github.com/user-attachments/assets/02de2034-c924-41e1-8237-0385b762e89e)

**Comment by kuba-04** [comment]
forgot to add the logs:

![Image](https://github.com/user-attachments/assets/4a136158-bfd0-44fc-91f2-edc215102b3c)

**Comment by kernelkind** [comment]
>  Have you seen an issue already open for this? if not, I can open a separate issue if you agree there is a bug, or maybe I misunderstand something here!

Yeah please make a new issue, I don't think we're tracking this yet
